build.cmd

```diff
@@ -1 +1 @@
-@powershell -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command "%~dp0\tools\run-csx.ps1 %~dp0\tools\build.csx %*"
+@dotnet run --project "%~dp0\tools\FakeItEasy.Build\FakeItEasy.Build.csproj"

```

Comment:
add a `-- %*` on the end of this so we don't always execute the default build.

Reply:
Oops! Yes, of course.